### PR TITLE
Added route parameterization

### PR DESCRIPTION
**Pull request by devang-gaur**
Also, renaming : measurments-api --> measurement-api

### CODE REVIEW

**Comment by anmolbabu** [comment]
Change all occurences of hdd to hypothesis if you are taking even that as part of this PR
also retain the name measurements instead of measurement(along with applicable prefixes and suffixes) @mitchharpur What do you suggest?
also all repo names would follow the name fabric8-hypothesis-<repo_name>-api-image and fabric8-hypothesis-<repo_name>-api is this also planned to be part of this PR or new PR?

File: app_constants.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 env_script_location=https://raw.githubusercontent.com/fabric8-hdd/hypothesis-core-image/master/cico_utils/setup_env.sh
-APP_NAME=openshift-hdd-measurements-api
-APP_PORT=$(grep "HDD_MEASUREMENTS_API_PORT" openshift/supervisor.env | cut -d'=' -f 2)
\ No newline at end of file
+APP_NAME=openshift-hdd-measurement-api
```
Comment (anmolbabu) [inline app_constants.sh]: osio-hypothesis-measurements-api

File: main.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 
 var port = process.env.APP_PORT || 9090
-var routes = require("./node_modules/measurements-api/src/api/routes.js")
+var routes = require("./node_modules/measurement-api/src/api/routes.js")
```
Comment (anmolbabu) [inline main.js]: var routes = require("./node_modules/measurements-api/src/api/routes.js")

**Comment by anmolbabu** [comment]
@dev-gaur Many places still refer to measurement-api instead of measurements-api please revisit this PR and fix them

**Comment by devang-gaur** [comment]
Apologies! Careless of me.
Changes made.

**Review by anmolbabu — APPROVED** [review]

**Pull request merged**